src/main/kotlin/com/exactpro/th2/check1/metrics/RuleMetric.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.check1.metrics
+
+import io.prometheus.client.Gauge
+
+object RuleMetric {
+    private val ACTIVE_TASK_COUNTER = Gauge.build("th2_check1_active_tasks_number", "Current active tasks number in execution").register()
```

Comment:
Let's add a bit of clarification to the help message. `execution` says nothing to me and it make the description hard to understand.
```suggestion
    private val ACTIVE_TASK_COUNTER = Gauge.build("th2_check1_active_tasks_number", "The number of currently working rules").register()
```
Or you can suggest another option to the description. The main point - it should help to understand what does the metric's value means.

Also, I was thinking about adding the label with rule type (CheckSequenceRule, CheckRule, etc). But maybe it is redundant for now

Reply:
I think it is a good idea to control which rules are working right now